src/code/audio_init_params.c

```diff
@@ -10,79 +10,97 @@ const s16 D_8014A6C0[] = {
     0x0030, // gTatumsPerBeat
 };
 
-const AudioContextInitSizes D_8014A6C4 = { 0x37F00, 0xE0E0, 0xBCE0 };
+// TODO: Extract from table?
+#define NUM_SOUNDFONTS 38
+#define SFX_SEQ_SIZE 0x6A90
+#define SFX_SOUNDFONT_1_SIZE 0x3AA0
+#define SFX_SOUNDFONT_2_SIZE 0x17B0
 
-ReverbSettings D_80133420[][3] = {
+// Sizes of everything on the init pool
+#define AI_BUFFERS_SIZE (AIBUF_LEN * sizeof(s16) * ARRAY_COUNT(gAudioContext.aiBuffers))
+#define SOUNDFONT_INFO_SIZE (NUM_SOUNDFONTS * sizeof(SoundFont))
+#define PERMANENT_POOL_SIZE (SFX_SEQ_SIZE + SFX_SOUNDFONT_1_SIZE + SFX_SOUNDFONT_2_SIZE)
+
+const AudioHeapInitSizes audioHeapInitSizes = {
```

Comment:
```suggestion
const AudioHeapInitSizes gAudioHeapInitSizes = {
```
how could you :japanese_goblin:

Reply:
Whoops